cheatsheets/gleam-for-python-users.md

```diff
@@ -755,6 +782,7 @@ An important difference to note is there is no OOP in Gleam. Methods can not be
 ### Unions
 
 In Python unions are declared using the `Union` type from the `typing module`
+or with the newer `|` syntax.
```

Comment:
Let's remove the reference to the `Union` and only mention the current `|` syntax.